CRM/Core/Payment/Form.php

```diff
@@ -129,8 +129,10 @@ protected static function addCommonFields(&$form, $paymentFields) {
       // This will cause the fields to be marked as required - but it is up to the payment processor to
       // validate it.
       $requiredPaymentFields[$field['name']] = $field['is_required'];
+      $paymentFieldDescriptions[$field['name']] = $field['description'];
     }
 
+    $form->assign('paymentFieldDescriptions', $paymentFieldDescriptions);
```

Comment:
Can we change this to
```
if (!empty($field['description'])) {
  $paymentFieldDescriptions[$field['name']] = $field['description'];
}
```

I feel there will be notice error when description is not present in $field.